package.json

```diff
@@ -12,6 +12,9 @@
 
   "dependencies": {
     "babel": "^5.8.23",
+    "bluebird": "^2.10.2",
```

Comment:
Because there are a lot of things that need to be updated to run on node v4 and we've only done sockit-to-me so far. :) So targeting 0.12 for now with the idea that v4 will come very shortly after that seems like a good strategy.